src/ContainerRegistry/ContainerRegistry/Commands/NewAzureContainerRegistry.cs

```diff
@@ -96,7 +101,8 @@ public override void ExecuteCmdlet()
                         Sku = new Microsoft.Azure.Management.ContainerRegistry.Models.Sku(Sku),
                         AdminUserEnabled = EnableAdminUser,
                         Tags = tags,
-                        Location = Location
+                        Location = Location,
+                        NetworkRuleSet = NetworkRuleSet?.GetNetworkRuleSet()
```

Comment:
I suggest we don't expose it from `create`, as we don't plan to GA this; and making it possible from `update` cmdlet is enough.